presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -39,4 +72,80 @@ private static String getStackTraceString(Throwable throwable)
         throwable.printStackTrace(new PrintWriter(errors));
         return errors.toString();
     }
+
+    private static void technicalDetailsRuntimeExceptionErrorMessage(StringBuilder builder, Throwable throwable, ClientSession session)
+    {
+        builder.append(TECHNICAL_DETAILS_HEADER);
```

Comment:
These constants are used only here, so I don't see the benefit of having them extracted. It makes it harder to tell exactly what's going to be printed, too. Inline them.